tests/fast-integration/kcp/client.js

```diff
@@ -12,16 +17,20 @@ class KCPConfig {
             getEnvOrThrow("KCP_TECH_USER_PASSWORD"),
             getEnvOrThrow("KCP_OIDC_CLIENT_ID"),
             getEnvOrThrow("KCP_OIDC_CLIENT_SECRET"),
+            getEnvOrDefault("KCP_MOTHERSHIP_API_URL", "https://mothership-reconciler.cp.dev.kyma.cloud.sap/v1"),
```

Comment:
Required by recent KCP-CLI; I thought it might make sense to fall back on *something*.

Reply:
I think it should throw an error regardless. For testing purposes we can add the env variable in the `proces.env.ENV_NAME = 'foo'` before invoking the command.

Reply:
Fine from my side!